nautobot/docs/user-guide/platform-functionality/echarts.md

```diff
@@ -0,0 +1,210 @@
+# ECharts
+
+This module is based on [Apache ECharts](https://echarts.apache.org/en/index.html). To create and configure charts in Nautobot, the `EChartsBase` class is used. It handles transforming input data, applying themes, and generating a valid ECharts option config JSON.
+
+## EChartsBase
+
+Base definition for an ECharts chart (no rendering logic). This class transforms input data, applies theme colors, and generates a valid ECharts option config. Currently, ECharts supports three types:
```

Comment:
Perhaps we need a `EChartsRaw` or similar, that does not transforms input data, but instead just handles the html for you. 

What say you @dsoftait and @glennmatthews

Reply:
Looks like that's handled already: https://github.com/nautobot/nautobot/pull/7726/files#diff-d1dbfeb8a45ab6886753f1981ae984d55a65f786b312201deff6f3a4e5debbbdR186-R188